plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -241,6 +241,8 @@ void chain_plugin::set_program_options(options_description& cli, options_descrip
           "Type of chaindb connection")
          ("chaindb_address", bpo::value<string>()->default_value("mongodb://127.0.0.1:27017"),
           "Connection address to chaindb")
+         ("genesis-file", bpo::value<bfs::path>(),
```

Comment:
This name is little confused with `genesis-json`. Maybe better call this option `genesis-data`. Then the user will have to specify two options when starting: `--genesis-json <...> --genesis-data <...>`. In my opinion it will be more intuitive.

Reply:
ok, I'll rename it in next PR